ghcup-0.0.7.yaml

```diff
@@ -4292,6 +4290,65 @@ ghcupDownloads:
             unknown_versioning:
               dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.10.1.0/cabal-install-3.10.1.0-armv7-linux-deb10.tar.xz
               dlHash: 79869dd72cb0a6f90f2e1e0e57af4ea6ee09b041550abc09252d1cf309b4f96e
+    3.10.2.0:
+      viTags:
+        - Latest
+        - Recommended
+      viChangeLog: https://github.com/haskell/cabal/blob/master/release-notes/cabal-install-3.10.2.0.md
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31020-64
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-alpine.tar.xz
+              dlHash: 9aac419fecb36f52339dca2ed9b1fac1c05b03628fb82d460f0e2cd895690280
+          Linux_Alpine:
+            unknown_versioning: *cabal-31020-64
+          Linux_CentOS:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-centos7.tar.xz
+              dlHash: 11496f7eb125be20c62a147a212f576d8794bd7597c002bc21d8c2c49171984d
+          Linux_Debian:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb10.tar.xz
+              dlHash: 545977a0faa44f366dc9b8184ae547ebf5dfc24c8a0a2c192295327b825fb12c
+          Linux_Fedora:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-fedora33.tar.xz
+              dlHash: 364681af04184dfae0a5d1e520ad0c3b4644936442e62383558bb8db09ccd617
+          Linux_Ubuntu:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-ubuntu20_04.tar.xz
+              dlHash: b3ae2f71d90af4bc76184fe4deb68dacf1ec67c5a0d2b9934f14955bc50195b5
+          Darwin:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-darwin.tar.xz
+              dlHash: f74864818f7fa388a94b7b0885af2959148e874e79b8acfb075bda60517333a0
+          Windows:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-windows.zip
+              dlHash: 67035900950c049e331b907492d1e07c0c540f20f72cf2df041093e2c1132f13
+          FreeBSD:
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.10.2.0/cabal-install-3.10.2.0-x86_64-freebsd12.tar.xz
```

Comment:
this returns 404

Reply:
yes I was meaning to ask you about the situation for the freebsd bindings: I have produced it myself, not through the official pipeline. I should put it in your official folder then?